### DISCUSSION TITLE
Select indivdual pages ?

### DISCUSSION THREAD

**Discussion by ZCaish1** [pdfarranger/pdfarranger#999] [Q&A]
I've been working with pdfarranger, 

It need an option to select individual images from the original pdf, not concurrent. 
Then export (it has export) but append to existing file, 
I scan dozens of documents into the master,  then would be nice to select individual images out of the pdf,  that are not in concurrent order, export them and append them to an existing file. 
Where now, I have to delete all the unneeded images out of the file, then save as. 
Also I can't seem to cut n paste another window of pdfarranger.

**Comment by jeromerobert** [comment]
You can already select individual and not concurrent pages (using ctrl) and export them to an intermediate file. You can then open the master file and append the intermediate file.

Concidering that load/save with PDF Arranger is a destructif operation (unfortunately), we will not implement "save as append to existing file".
